numpy/random/_generator.pyx

```diff
@@ -496,11 +493,9 @@ cdef class Generator:
             Output shape.  If the given shape is, e.g., ``(m, n, k)``, then
             ``m * n * k`` samples are drawn.  Default is None, in which case a
             single value is returned.
-        dtype : {str, dtype}, optional
-            Desired dtype of the result. All dtypes are determined by their
-            name, i.e., 'int64', 'int', etc, so byteorder is not available
-            and a specific precision may have different C types depending
-            on the platform. The default value is `np.int_`.
+        dtype : dtype, optional
+            Desired dtype of the result. Byteorder must be native.
+            The default value is `np.int_`.
```

Comment:
The default (line 469) is np.int64 but default is documented as np.int_? That seems wrong.

Reply:
Are there more mistakes like this across the docstrings?

Reply:
updated